packages/drivers/routerlicious-driver/src/test/restWrapper.spec.ts

```diff
@@ -8,6 +8,7 @@ import { TelemetryUTLogger } from "@fluidframework/telemetry-utils";
 import { DriverErrorType } from "@fluidframework/driver-definitions";
 import { RateLimiter } from "@fluidframework/driver-utils";
 import nock from "nock";
+import { ITokenResponse } from "@fluidframework/routerlicious-driver-previous";
```

Comment:
This is probably an unintentional import from driver-previous. Guessing you meant to import from "../tokens"?